packages/backend/src/plugins/catalog.ts

```diff
@@ -26,6 +27,18 @@ export default async function createPlugin(
     }),
   );
 
+  builder.addEntityProvider(
+    GithubOrgEntityProvider.fromConfig(env.config, {
+      id: 'production',
+      orgUrl: 'https://github.com/raf-backstage-demo',
+      logger: env.logger,
+      schedule: env.scheduler.createScheduledTaskRunner({
+        frequency: { minutes: 60 },
+        timeout: { minutes: 15 },
```

Comment:
Including the  initalDelay parameter for syncing helps

Reply:
is there any change you need here?

Reply:
@riginoommen

Reply:
@riginoommen how would it help? I mean, why would waiting be a good thing? Also what value do you want for the initialDelay?

Reply:
done